CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+# dbt_facebook_ads_source v0.7.3
+[PR #35](https://github.com/fivetran/dbt_facebook_ads_source/pull/35) includes the following updates:
+
+## Bug Fixes
+- Resolved the "duplicate column" error that would arise when the `facebook_ads__basic_ad_passthrough_metrics` variable included `reach` or `frequency`. `Reach` and `frequency` are included by default in the `stg_facebook_ads__basic_ad` model but will only persist to downstream transform models if specified in the `facebook_ads__basic_ad_passthrough_metrics` variable.
```

Comment:
Should we call out how we resolved it in a sub-bullet using a for loop in the specific models/macros? Doesn't have to be too involved, just always nice to explain how to the customer.

Reply:
added!